ext/js/display/display-anki.js

```diff
@@ -263,7 +263,9 @@ export class DisplayAnki {
 
     /** */
     _onContentUpdateComplete() {
-        void this._updateDictionaryEntryDetails();
+        if (this._display.getOptions()?.anki.enable) {
```

Comment:
Can we put this check inside `_updateDictionaryEntryDetails`? If there are other events that require calling `_updateDictionaryEntryDetails` I feel like we'll have to duplicate this check in multiple places